homeassistant/components/conversation/default_agent.py

```diff
@@ -147,6 +149,29 @@ def get_or_load_intents(self, language: str) -> LanguageIntents | None:
 
                 # Will need to recreate graph
                 intents_changed = True
+                _LOGGER.info(
+                    "Loaded intents component=%s, language=%s", component, language
+                )
+
+            # Check for custom sentences
+            custom_sentences_path = Path(
+                self.hass.config.path("custom_sentences", component, f"{language}.yaml")
```

Comment:
We should look at `<custom_sentences>/language/` and load all the YAML files. That way people can have multiple language packs from different people.